### DISCUSSION TITLE
blog/inversion-of-control-and-dependency-injection-in-java

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#81] [Announcements]
# blog/inversion-of-control-and-dependency-injection-in-java

This blog post explains Inversion of Control and Dependency Injection in Java. These concepts are the backbone of modern enterprise Java frameworks.

https://www.cesarsotovalero.net/blog/inversion-of-control-and-dependency-injection-in-java.html

**Comment by magantisurya** [comment]
Excellent article on Inversion of control in Spring framework
